specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2017-07-01/examples/AzureIaasVm/ValidateOperation_RestoreDisk.json

```diff
@@ -0,0 +1,48 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "testRG",
+    "vaultName": "testVault",
+    "fabricName": "Azure",
+    "containerName": "IaasVMContainer;iaasvmcontainerv2;testRG;testvmName",
+    "protectedItemName": "VM;iaasvmcontainerv2;testRG;testvmName",
+    "recoveryPointId": "348916168024334",
+    "api-version": "2017-07-01",
+    "parameters": {
+      "objectType": "ValidateIaasVMRestoreOperationRequest",
+      "restoreRequest": {
+        "recoveryPointId": "348916168024334",
+        "objectType": "IaasVMRestoreRequest",
+        "recoveryType": "RestoreDisks",
+        "sourceResourceId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/netsdktestrg/providers/Microsoft.Compute/virtualMachines/netvmtestv2vm1",
+        "storageAccountId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/testingRg/providers/Microsoft.Storage/storageAccounts/testAccount",
+        "region": "southeastasia",
+        "createNewCloudService": true,
+        "originalStorageAccountOption": false,
+        "api-version": "2016-12-01",
```

Comment:
OK regarding the model validation failures, I dug into this some more and the only problem is this extra "api-version" parameter in the request body.  The reporting of the other properties appears to be a bug in the validation tool and we are investigating futher.